packages/kbn-optimizer/src/common/rxjs_helpers.test.ts

```diff
@@ -70,12 +70,13 @@ describe('maybeMap()', () => {
 });
 
 describe('debounceTimeBuffer()', () => {
+  const delay = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
   beforeEach(() => {
-    jest.useFakeTimers();
+    jest.useRealTimers();
   });
 
   afterEach(() => {
-    jest.useRealTimers();
+    jest.useFakeTimers();
```

Comment:
Why is this necessary? I feel like the `fakeSchedulers` approached used in the optimizer would be better than relying on sleeps.

Reply:
The only reason is that I "fixed" this test before finding out about the `fakeScheduler` helper. I'll change it tomorrow morning.

Reply:
Done :)